README.md

```diff
@@ -32,14 +32,15 @@ Join [our PostHog Users Slack community](https://join.slack.com/t/posthogusers/s
 
 ## Features
 
-- **Event-based** analytics at a user level - see which users are doing what in your application.
+- **Event-based** analytics at a user level - capture your product's usage data to see which users are doing what in your application.
+- **Product data visualizations**: [graphs](https://posthog.com/docs/features/trends), [funnels](https://posthog.com/docs/features/funnels), [user cohorts](https://posthog.com/docs/features/cohorts), [user paths](https://posthog.com/docs/features/paths), [retention tables](https://posthog.com/docs/features/retention), and [dashboards](https://posthog.com/docs/features/dashboards).
 - **Complete control** over your data - host it yourself.
-- **Automatically capture** clicks and page views to do analyze what your users are doing **retroactively**.
+- **Session recording** to [watch videos](https://posthog.com/docs/features/session-recording) of your user behavior, with fine-grained privacy controls.
+- **Automatically capture** [clicks and pageviews](https://posthog.com/docs/features/actions) to do analyze what your users are doing **retroactively**.
 - **Feature flags** to understand the impact of new features before rolling them out more widely
-- **Heatmaps** to understand where in your app or website people are clicking
-- **Ready-made libraries** for **[JS](https://posthog.com/docs/integrations/js-integration), [Python](https://posthog.com/docs/integrations/python-integration), [Ruby](https://posthog.com/docs/integrations/ruby-integration), [Node](https://posthog.com/docs/integrations/node-integration), [Go](https://posthog.com/docs/integrations/go-integration)** + [API](https://posthog.com/docs/integrations/api) for anything else.
-- **Beautiful data visualizations**: [graphs](https://posthog.com/docs/features/trends), [funnels](https://posthog.com/docs/features/funnels), [user cohorts](https://posthog.com/docs/features/cohorts), [user paths](https://posthog.com/docs/features/paths), [retention tables](https://posthog.com/docs/features/retention), and [dashboards](https://posthog.com/docs/features/dashboards).
-- **Analytics right on your website** with the playful [PostHog Toolbar](https://posthog.com/docs/features/toolbar).
+- **Heatmaps** with the [Toolbar](https://posthog.com/docs/features/toolbar).
```

Comment:
I'd use 'PostHog Toolbar'

Reply:
Genuine q: We don't have 'PostHog Funnels', why would we label the toolbar like this?

Reply:
then maybe decapitalize it - since having the capital T implies some exclusive thing e.g. we don't capitalize funnels (although I think In was the one who capitalized this a while back?)

But yeah my rationale is that funnels every platform has, whereas this is kind of unique to us. "heatmaps with the toolbar" sounds like I should know that toolbars do heatmaps whereas "PostHog Toolbar" here signals to me that this is a PostHog-specific thing.

Super minor tho